MusicNotationKit/MusicNotationKit/Tuplet.swift

```diff
@@ -200,6 +481,11 @@ public struct Tuplet: NoteCollection {
 
 extension Tuplet: Equatable {
     public static func ==(lhs: Tuplet, rhs: Tuplet) -> Bool {
+        guard lhs.noteTimingCount == rhs.noteTimingCount &&
+            lhs.noteDuration == rhs.noteDuration &&
+            lhs.noteCount == rhs.noteCount else {
+                return false
+        }
         guard lhs.notes.count == rhs.notes.count else {
```

Comment:
The way it collapsed it, it doesn't show, but we are already comparing each note.